day_7/caesar_cipher.rb

```diff
@@ -0,0 +1,23 @@
+# Caesar ciphered
+
+def caesar_cipher
+  # Print and get user_string input
+  puts "Welcome to Caesar Cipher where you can encode whatever you like."
+  puts "Enter a phrase or word you wish to encode:"
+  print "> "
+  user_string = ($stdin.gets.chomp).downcase
+  # Print and get shifts input
+  puts "Ok, please select a number between to determine how it will be encoded"
+  shifts = (gets.chomp).to_i
+  # Create alphabet array
+  alphabet = Array('a'..'z')
```

Comment:
Nice- Love that you leveraged the processing power of your computer over hard coding a whole alphabet array. Also, very nice work explaining individual lines of code.